src/model/impl/TaskManager.cpp

```diff
@@ -167,6 +167,28 @@ std::vector<TaskDTO> TaskManager::ShowChild(const TaskId& parent_id)
     }
     return result;
 }
+std::vector<TaskDTO> TaskManager::ShowTasksWithLabel(const std::string& label)
+{
+    auto result = std::vector<TaskDTO>{};
+
+    for (const auto& elem: this->tasks_)
+    {
+        auto task = elem.second.GetTask();
+        auto has_label = std::count_if(task.labels().cbegin(), task.labels().cend(),
```

Comment:
Why `std::count_if`? Maybe `std::find` is better option here?

Reply:
Fixed